pyEpiabm/pyEpiabm/tests/test_unit/test_core/test_microcell.py

```diff
@@ -82,6 +82,19 @@ def test_set_location(self):
         self.assertRaises(ValueError, microcell.set_location, (1, (8, 6)))
         self.assertRaises(ValueError, microcell.set_location, ('1', 1))
 
+    def test_count_icu(self):
+        self.microcell.add_people(6)
+        for person in self.microcell.persons:
+            person.update_status(InfectionStatus(7))
+        self.assertEqual(self.microcell.count_icu(), 6)
+
+    def test_count_infectious(self):
+        self.microcell.add_people(6)
+        for i in range(6):
+            person = self.microcell.persons[i]
+            person.update_status(InfectionStatus(i+3))
+        self.assertEqual(self.microcell.count_infectious(), 6)
```

Comment:
```suggestion
        self.microcell.add_people(6)
        for i in range(4):
            person = self.microcell.persons[i]
            person.update_status(InfectionStatus(i+3))
        self.assertEqual(self.microcell.count_infectious(), 4)
```
Can you think why this change might be better practice?